.github/workflows/codegen.yml

```diff
@@ -14,6 +14,7 @@ on:
 jobs:
   codegen:
     runs-on: ubuntu-18.04
+    if: github.repository == 'rook/rook'
```

Comment:
this change is for testing only.

Reply:
https://github.com/rook/rook/pull/7942/checks?check_run_id=2617453076 form this we can confirm it is not skipped which was happening last time.

Reply:
@subhamkrai Can we confirm that it will run in the `rook/rook` fork with this condition? For example, if you change it to your fork to test, do you see it run?

Reply:
@travisn I tested in fork it didn't run https://github.com/subhamkrai/rook/actions/runs/855834143